pyproject.toml

```diff
@@ -36,9 +36,11 @@ lines_between_types = 1
 
 [tool.towncrier]
 package = "plasmapy"
+name = "PlasmaPy"
 filename = "CHANGELOG.rst"
 directory = "changelog/"
-issue_format = "`#{issue} <https://github.com/plasmapy/plasmapy/pull/{issue}>`__"
+issue_format = ":issue:`{issue}`"
+wrap = true
```

Comment:
Do you want to add back the `title_format` key so the changelog titles have the version `v` and look like `PlasmaPy v0.8.2.dev...` instead of `PlasmaPy 0.8.2.dev...`?

Reply:
Good call!  I missed that the `v` was missing.